### ISSUE TITLE
Multi select and multi check

### ISSUE THREAD

**Issue by senna23** [URUWorks/TeroSubtitler#333]
Multi select at a time with "shift" on keyboard, and then multi check:
![image](https://github.com/user-attachments/assets/f3ad54f4-14bb-4776-bfcc-6fa7985dad64)

**Comment by URUWorks** [comment]
It's there, right click and the context menu appears.

**Comment by chenlung** [comment]
> It's there, right click and the context menu appears.
> You can also check and uncheck with the space bar.

Yes, you’re right about multi-check, but multi-select isn’t an option (holding shift and clicking the top and middle to select this and in between). I don’t know how valid this is?
